### DISCUSSION TITLE
🔗 Active styles for pagination in Loop Carousel widget

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21329] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I didn't pay attention to this in the beta version. But in the pagination of the loop carousel widget, it is not possible to change the pagination color in active mode. This is really a big limitation.
Currently, it is only possible to customize the color in normal and hover modes : 

![a](https://user-images.githubusercontent.com/88034902/218723886-5ef00247-c7d3-4a87-92d7-38549345baba.png)

Please add customization option in active mode.Even old widgets like slides have this possibility. We are now experiencing 2023 widgets.

### Describe the solution you'd like

Please add active mode to loop carousel widget pagination : 

![v](https://user-images.githubusercontent.com/88034902/218725620-73400eba-6e30-4ae6-8e42-21ebf452278f.png)

**Comment by FabioMontenegro** [comment]
In addition, it will be great add numbers in pagination.
